assets/stylesheets/components/_page-footer.scss

```diff
@@ -0,0 +1,3 @@
+.page-footer {
+  @include clearfix;
```

Comment:
This declaration is too opinionated – we might want `.page-footer` elsewhere later without the clearfix treatment. 

Since [Bootstrap already provides](https://github.com/twbs/bootstrap-sass/blob/5d6b2ebba0c2a5885ce2f0e01e9218db3d3b5e47/assets/stylesheets/bootstrap/_utilities.scss#L9-L11) a `.clearfix` selector to do this exact thing... please update your implementation in crdschurch/crds-styleguide#36 to use that selector instead.